src/EnergyPlus/FanCoilUnits.cc

```diff
@@ -1624,7 +1677,7 @@ namespace FanCoilUnits {
 				// get full load result
 				mdot = FanCoil( FanCoilNum ).MaxColdWaterFlow;
 				SetComponentFlowRate( mdot, FanCoil( FanCoilNum ).ColdControlNode, FanCoil( FanCoilNum ).ColdPlantOutletNode, FanCoil( FanCoilNum ).CWLoopNum, FanCoil( FanCoilNum ).CWLoopSide, FanCoil( FanCoilNum ).CWBranchNum, FanCoil( FanCoilNum ).CWCompNum );
-				Calc4PipeFanCoil( FanCoilNum, ControlledZoneNum, FirstHVACIteration, QUnitOutMaxHC );
+				Calc4PipeFanCoil( FanCoilNum, ControlledZoneNum, FirstHVACIteration, QUnitOutMaxHC, _, ElectricHeaterControl );
```

Comment:
@rraustad Doesn't seem like an extra argument should be necessary here.  Can't Calc4PipeFanCoil figure out the situation on it's own?